src/prettify.tsx

```diff
@@ -0,0 +1,7 @@
+export type Prettify<T> = T extends object
+  ? {
+      [K in keyof T]: T[K];
+    } & {
+      /** */
+    }
+  : T;
```

Comment:
what does this file do?

Reply:
https://www.totaltypescript.com/concepts/the-prettify-helper

I thought it'd work on those big types, but it doesn't, unfortunately. I forgot to delete it.